custom/src/main/java/co/elastic/otel/ElasticBreakdownMetrics.java

```diff
@@ -1,301 +0,0 @@
-/*
```

Comment:
[for reviewer] removed because it's unused and we'll re-implement it later if needed